test/Libraries/RevitIntegrationTests/SubelementTests.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using NUnit.Framework;
+using Revit.Elements;
+using RevitTestServices;
+using RTF.Framework;
+using Autodesk.Revit.DB;
+using Autodesk.Revit.DB.Analysis;
+
+namespace RevitSystemTests
+{
+    [TestFixture]
+    class SubelementTests : RevitSystemTestBase
+    {
+        private const Int64 m_testElbowId = 976586L;
+
+        [Test]
+        [TestModel(@".\Subelement\DuctSegmentFlow.rvt")]
+        public void CanSetCoefficientOnDuctSegment()
```

Comment:
This looks like more close to an API/unit test, which is fine, but you don't even use the SubelementTest.dyn Dynamo script you added, will there be another test added?

Reply:
Indeed, this is an API test. The attached script utilized the overload methods to get/set the subelement parameter values. There is also another test in Revit repo.